dashboard/config/locales/dsls.zh-CN.yml

```diff
@@ -644,11 +644,11 @@ zh-CN:
 
           ## 已学代码
 
-          * **[HTML](//docs.code.org/weblab/html/)** - `<html></html>`
-          * **[Doctype](//docs.code.org/weblab/doctype/)** - `<!DOCTYPE>`
-          * **[Paragraph](//docs.code.org/weblab/P/)** - `<p></p>`
-          * **[Body](//docs.code.org/weblab/body/)** - `<body></body>`
-          * **[Head](//docs.code.org/weblab/head/)** - `<head></head>`
+          * **[HTML](//studio.code.org/docs/weblab/html/)** - `<html></html>`
+          * **[Doctype](//studio.code.org/docs/weblab/doctype/)** - `<!DOCTYPE>`
+          * **[Paragraph](//studio.code.org/docs/weblab/P/)** - `<p></p>`
+          * **[Body](//studio.code.org/docs/weblab/body/)** - `<body></body>`
+          * **[Head](//studio.code.org/docs/weblab/head/)** - `<head></head>`
```

Comment:
@Hamms , we're trying to update a url that's embedded in some translations, and believe we have updated all the english locations properly. can you confirm that this approach of updating the per-language message files will suffice to fix it for other languages, or is there some chance the old url will reappear here and undo his when the next i18n sync happens?

Reply:
You're right to be concerned! I noted this in https://github.com/code-dot-org/code-dot-org/pull/32487#discussion_r360004265